templates/common/_base/units/ovsdb-server.service.yaml

```diff
@@ -3,5 +3,8 @@ enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else if eq .NetworkType "Op
 dropins:
   - name: 10-ovsdb-restart.conf
     contents: |
+      [Unit]
+      After=sysroot.mount
```

Comment:
I think what you really want is `After=ostree-remount.service`.

But again this is just a workaround only necessary on older ostree, xref https://bugzilla.redhat.com/show_bug.cgi?id=2066964#c20